01week/rockPaperScissors.js

```diff
@@ -7,17 +7,40 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
+const rockPaperScissors=(hand1, hand2)=> {
+  const aTie = "Its's a tie!";
```

Comment:
Since you only use this string once there's no need to use memory on it; just return the string.